test/unit/lib/utils/gpDataMapper.js

```diff
@@ -4,23 +4,31 @@ const gpDataMapper = require('../../../../app/lib/utils/gpDataMapper');
 const expect = chai.expect;
 
 describe('GP Surgery Data Mapper', () => {
-  it('for a GP Surgery that has POMI data and a url will return the url', () => {
-    const gpData = { bookingSystem: { supplier: 'EMIS', bookOnlineLink: 'http://example.com' } };
+  it('for a GP Surgery that has appointments POMI data and a url will return the url', () => {
+    const gpData = { onlineServices: { appointments: { supplier: 'EMIS', url: 'http://example.com' } } };
     const expectedResult = 'http://example.com';
     const result = gpDataMapper(gpData);
 
     expect(result).to.be.equal(expectedResult);
```

Comment:
I think the pattern makes reading what the expected result is unnecessarily opaque. I see no value in having it as a variable, it adds an extra line and I don't see them being reused. I feel strongly enough that this change is worth making. No code is good code.